### DISCUSSION TITLE
No hint path defined for [backpack]. when show abort in public page

### DISCUSSION THREAD

**Discussion by christmex** [Laravel-Backpack/community-forum#573] [Q&A (Help)] [answered]
Hi guys, I have a question, so i want to show the error page not in backpack panel, but for public page

I have this case as an example
```php
//web.php inside routes folder
Route::get('/', function () {
  return abort(403,'No access');
}
```

it shows, this error
<img width="532" alt="image" src="https://github.com/Laravel-Backpack/community-forum/assets/34405837/d66f2008-a61c-458d-89aa-dd33efc8bbc9">

**What did I do**
I already take a look at this issue https://github.com/Laravel-Backpack/community-forum/discussions/554
but I'm not found anything to solve this error, sorry if I miss something

**what I want**
show the error page

please help me out, i can solve this with show them a custom blade file to handle this, but I want to know why the abort function does not work thanks guys 🙏

**Comment by tabacitu** [comment, marked as answer]
The reason you're no longer seeing Backpack error pages in non-admin-panel pages is... because we fixed it 😀 It usually doesn't make sense to have the admin panel design dictate something in the app itself, not even just error views. So in v6 the error pages have been moved, they're no longer in `resources/views/errors`.

You can still show Backpack error pages in custom pages, but ONLY if they're in the admin panel (under the admin panel URL prefix). To us, that's the only place where they make sense.

If you liked the previous behaviour where app errors looked like Backpack errors, you should be able to copy them from vendor to your `resources/views/errors` and fix the layout a bit, to have them again.

Hope it helps.
Cheers!

**Reply by christmex** [reply]
@tabacitu Thx 🙏
